plugin/trino-accumulo/src/test/java/io/trino/plugin/accumulo/TestAccumuloConnectorTest.java

```diff
@@ -296,6 +296,13 @@ public void testShowCreateTable()
                         ")");
     }
 
+    @Test
+    @Override
+    public void testCreateViewSchemaNotFound()
+    {
+        // TODO (https://github.com/trinodb/trino/issues/12475) Accumulo connector can create new views in a schema where it doesn't exist
```

Comment:
Please use `assertThatThrownBy()` to catch future change.

Reply:
ok